src/AwesomeCMSCore/Modules/AwesomeCMSCore.Modules.Account/Controllers/API/GroupController.cs

```diff
@@ -0,0 +1,29 @@
+﻿using AwesomeCMSCore.Modules.Account.Services;
+using AwesomeCMSCore.Modules.Entities.Entities;
+using Microsoft.AspNetCore.Authentication.JwtBearer;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace AwesomeCMSCore.Modules.Account.Controllers.API
+{
+    [Authorize(AuthenticationSchemes = JwtBearerDefaults.AuthenticationScheme)]
+    [Route("api/[controller]/[action]")]
+    public class GroupController : Controller
+    {
+        private readonly IGroupService groupService;
+        public GroupController(IGroupService group)
```

Comment:
groupService

Reply:
??? @ngohungphuc

Reply:
change to public GroupController(IGroupService groupService)
Is just for consistent

Reply:
Okay